packages/augur-core/src/contracts/AugurWalletRegistryV2.sol

```diff
@@ -51,7 +51,7 @@ contract AugurWalletRegistryV2 is Initializable, BaseRelayRecipient, Forwarder {
     uint256 private constant POST_RELAY_GAS_COST = 200000;
 
     // Gas stipends for acceptRelayedCall, preRelayedCall and postRelayedCall
-    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 50000;
+    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 75000;
```

Comment:
tldr on why this needs to be bumped?

Reply:
The signature validation was removed as part of the standard relayHub validation and made incumbent upon the recipient to validate in this call, so we have to do more in it.